cekit/templates/template.jinja

```diff
@@ -74,7 +74,8 @@ rm{% for repo in repositories %} /etc/yum.repos.d/{{ repo.filename }}{% endfor %
  #}
 {%- macro repo_clear_cache(pkg_manager) -%}
     {% if pkg_manager in helper.SUPPORTED_PACKAGE_MANAGERS %}
-{{ pkg_manager }} clean all && [ ! -d /var/cache/yum ] || rm -rf /var/cache/yum
+{{ pkg_manager }} clean all \
```

Comment:
Do we really want `clean all` still here?

Reply:
No, I'll drop it. Thanks